src/index.ts

```diff
@@ -8,9 +8,22 @@ import { arrayHelper } from './utils/arrayHelper';
 import { colorHelper } from './utils/color';
 import { stringHelper } from './utils/stringHelper';
 import { disposable } from './utils/usingDisposable';
+
+/**
+ * Use named exports here.
+ */
+export * from './hooks';
 export * from './types';
+export * from './utils';
+export { arrayHelper } from './utils/arrayHelper';
+export { colorHelper } from './utils/color';
+export * from './utils/deviceInfo/deepDeviceInfo';
+export * from './utils/deviceInfo/deviceInfo';
+export { stringHelper } from './utils/stringHelper';
+export { disposable } from './utils/usingDisposable';
```

Comment:
`arrayHelper`, `colorHelper`, `stringHelper`, `disposable` is already exported as part of the default export, `EchoUtils` object.

Are these redundant named exports really needed?

Reply:
I forgot to write about this in the description. I propose that the stuff from Echo Utils should be named exported so that things are more easily imported elsewhere. Today, you have to import the object and then retrieve what you need from it. With named exports, you just do a destructured import of whatever you need and away you go.

![image](https://user-images.githubusercontent.com/10920843/234618169-7fef42dc-dd5c-40a6-a0e5-71f03b18cbbb.png)

Reply:
Yes, I see the benefit of this approach as well.

I guess this is more a question of preference: the default import was introduced with the intention of better organize the exported things.

If one doesn't know about "deviceInfo", how do you know what to import from `echo-utils`? But if you import EchoUtils, and start to type, intellisense will show you the categories, and (hopefully) it's easier to find what one is looking for.

I see that this approach has it's own pros and cons.

I don't have strong opinion about this, so it's ok from my side to have the named exports as well.